portfolio/src/main/webapp/script.js

```diff
@@ -246,3 +246,29 @@ function filter(location) {
 
 // Append the "mapscript" element to "head"
 document.head.appendChild(mapscript);
+
```

Comment:
Can we separate functions for different feature into different js files?

Reply:
Sure!